src/pages/Account/Questions.tsx

```diff
@@ -49,28 +83,36 @@ const Questions: React.FunctionComponent<PropsType> = ({ location }) => {
     updateTheme(venue);
   }, [venue]);
 
-  if (!venue) {
-    return <>Loading...</>;
+  if (!venueId) {
+    return <>Error: Missing required venueId param</>;
   }
 
-  // Skip this screen if there are no profile questions for the venue
-  if (!venue?.profile_questions?.length) {
-    proceed();
+  if (isLoaded(venue) && !venue) {
+    return <>Error: venue not found for venueId={venueId}</>;
+  }
+
+  if (!venue) {
+    return <LoadingPage />;
   }
 
   const headerMessage =
     "To help other Hubbers find and get to know you, tell us a little about yourself.";
   const REQUIRED_QUESTIONS_NUMBER = 1;
 
   return (
-    <div className="page-container questions-container">
-      <div className="hero-logo github"></div>
+    <div className="Questions page-container questions-container">
+      <div className="hero-logo github" />
       <div className="login-container">
         <h2 className="header-message">{headerMessage}</h2>
+
+        <p className="subheader-message">
+          This will help your fellow partygoers break the ice
+        </p>
+
```

Comment:
It looks like this might have been removed from `env/github` since it's showing as 'newly added' here, yet not in https://github.com/sparkletown/sparkle/pull/1754